index.js

```diff
@@ -26,12 +26,9 @@ app.use(updateRouter)
 app.use(marketplaceRouter)
 app.use(commentRouter)
 app.use(express.static(__dirname + '/public'));
-
-//setting port
-let port = process.env.PORT
-if (port == null || port == ""){
-    port = 3000;
-}
-app.listen(port, () => {
-    console.log(`Server running on port ${port}`)
```

Comment:
This code needs to be turned back, `process.env.PORT` is imp for deployment to heroku